app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :users, foreign_key: :author_id, inverse_of: :comments
+  belongs_to :posts, inverse_of: :comments
```

Comment:
- [ ] Similarly to Likes as discussed above:
       1. A comment should belong to an `author` according to the ERD, not `users`.
       2. A comment should belong to one `post` not `posts`.
       
     Please refactor as appropriate.